.changeset/hungry-foxes-divide.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-catalog-graph': patch
+---
+
+Expose props `renderNode` & `renderLabel` to Catalog Graph Page
```

Comment:
We should update this to include the inclusion of all props from the underlying graph not just these 2.